ports/geckolib/glue.rs

```diff
@@ -343,6 +343,19 @@ pub extern "C" fn Servo_GetComputedValuesForPseudoElement(parent_style: *mut Ser
 }
 
 #[no_mangle]
+pub extern "C" fn Servo_GetComputedValuesForOtherNonElement(parent_style: *mut ServoComputedValues)
+     -> *mut ServoComputedValues {
+    type Helpers = ArcHelpers<ServoComputedValues, GeckoComputedValues>;
+    Helpers::with(parent_style, |parent| {
+        // Non-elements such as placeholder frames, like text nodes, don't match
+        // any rules, and so just inherit all inherited properties and have
+        // initial values for non-inherited properties.
+        let style = GeckoComputedValues::style_for_child_text_node(parent);
```

Comment:
And then I think we should impl a method on GeckoComputedValues::inherit that does this, and have style_for_child_text_node delegate to that.